contracts/terraswap_factory/Cargo.toml

```diff
@@ -34,15 +34,16 @@ overflow-checks = true
 backtraces = ["cosmwasm-std/backtraces"]
 
 [dependencies]
-classic_terraswap = { path = "../../packages/classic_terraswap", default-features = false, version = "2.6.1"}
-cosmwasm-std = { version = "0.16.7" }
-cw-storage-plus = { version = "0.8.0" } 
+classic-bindings = { version = "0.1.1" }
+cosmwasm-std = { version = "1.0.0" }
+cw-storage-plus = { version = "0.12" } 
```

Comment:
0.12 support to 1.0.0-beta. So need to change v0.13.4 or higher

Reply:
https://github.com/terraswap/classic-terraswap/pull/14/commits/02d389e85a7a6b948445b233d591cb2bfb0e816a